src/test/com/arrary/Arraylist.java

```diff
@@ -0,0 +1,32 @@
+package test.com.arrary;
```

Comment:
what does arrary mean? And why do you do it in the test folder? You don't use @Test annotations so what is the motivation for it?